packages/odie-client/src/data/use-send-odie-message.ts

```diff
@@ -296,7 +296,6 @@ export const useSendOdieMessage = ( signal: AbortSignal ) => {
 			}
 		},
 		onSettled: () => {
-			setChatStatus( 'loaded' );
```

Comment:
this was added in [here](https://github.com/Automattic/wp-calypso/commit/922f46b80c49838c4c747ecf91c756e1a6917b04#diff-ba88a18a0a9dd3acfe514e48c9f94d1ef197b1fd5436ca4a94f044ac39f84797R235)
and it switches the chat status from `transfer` to `loaded`